sopel/cli/config.py

```diff
@@ -167,6 +188,78 @@ def handle_get(options):
     return 0  # successful operation
 
 
+def handle_set(options):
+    """Set the ``<section> <key>`` setting to ``<value>``.
+
+    :param options: parsed arguments
+    :type options: :class:`argparse.Namespace`
+    :return: 0 if everything went fine;
+             1 if the section and/or key does not exist;
+             2 if the settings can't be loaded
+    """
+    try:
+        settings = utils.load_settings(options)
+    except Exception as error:
+        tools.stderr(error)
+        return 2
+
+    section = options.section
+    option = options.option
+    value = options.value
+
+    # Make sure the section exists
+    if not settings.parser.has_section(section):
+        try:
+            settings.parser.add_section(section)
+        except (TypeError, ValueError):
+            tools.stderr('Invalid section name "%s"' % section)
+            return 1
+
+    # Update the option & save config file
+    settings.parser.set(section, option, value)
```

Comment:
Does that validate the value type?

Reply:
Hmm, I guess not.

![image](https://user-images.githubusercontent.com/164140/98574923-27917580-227e-11eb-82ce-e8dee2ca0fd3.png)

That should be validated as `int`. Guess I should copy what `admin.py` does with `setattr()` stuff.

Reply:
I'll mark this as resolved-ish (meaning it's As Good As It's Going To Get without a major rework of some internals; see commit message for d49b679).